javascript/MLE-Demo/cleanup.sql

```diff
@@ -0,0 +1,5 @@
+BEGIN
+  EXECUTE IMMEDIATE 'DROP PROCEDURE SET_WALLET';
+  EXECUTE IMMEDIATE 'DROP TABLE INVENTORY';
+  EXCEPTION WHEN OTHERS THEN NULL;
```

Comment:
In the `setup.sql` we also add specific ACLs which need to be revoked again on clean up.